render/renderer.go

```diff
@@ -0,0 +1,1071 @@
+/*
+ * This file is subject to the terms and conditions defined in
+ * file 'LICENSE.md', which is part of this source code package.
+ */
+
+package render
+
+import (
+	"errors"
+
+	"github.com/adrg/sysfont"
+	"github.com/unidoc/unipdf/v3/common"
+	pdfcontent "github.com/unidoc/unipdf/v3/contentstream"
```

Comment:
Use contentstream?  I generally prefer to use the actual package name, only aliases where necessary due to naming conflicts.  If package name is not good, we can consider addressing that in future major version.

Reply:
https://github.com/unidoc/unipdf/wiki/UniPDF-Developer-Guide#19-import-group-styles